drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -68,7 +68,7 @@
 /* Timing (all in units of microseconds) */
 
 #define MMCSD_POWERUP_DELAY     ((useconds_t)250)    /* 74 clock cycles @ 400KHz = 185uS */
-#define MMCSD_IDLE_DELAY        ((useconds_t)50000)  /* Short delay to allow change to IDLE state */
+#define MMCSD_IDLE_DELAY        ((useconds_t)100000)  /* Short delay to allow change to IDLE state */
```

Comment:
align the comment

Reply:
@SimonFilgis let's squash two patch into one:
git rebase --interactive HEAD~2
git push -f origin cherrypick_sd_card_timing